packages/components/src/NavBar/NavBar.tsx

```diff
@@ -101,54 +130,90 @@ const UserAvatar = styled.figure`
+        <NavInner>
+          <NavLink to="/registry">
+            <NavLinkRegistryText />
           </NavLink>
-          <NavLink href="https://civil.co/how-to-launch-newsroom/" target="_blank">
-            How to launch a newsroom
+          <NavLink to="/parameterizer">
+            <NavLinkParameterizerText />
           </NavLink>
-          <NavLink href="https://civil.co/white-paper/" target="_blank">
-            White Paper
+          <NavLink to="/createNewsroom">
+            <NavLinkCreateNewsroomText />
           </NavLink>
-        </NavDropDown>
-        <NavAccent>
-          <NavLink to="/dashboard">My Activity</NavLink>
-        </NavAccent>
-        <NavUser>
-          <CvlContainer>
-            <CvlToken />
-            <span>
-              <UserCvlBalance>{props.balance}</UserCvlBalance>
-              <UserCvlVotingBalance>{"+" + props.votingBalance}</UserCvlVotingBalance>
-            </span>
-          </CvlContainer>
-          <AvatarContainer>
-            <UserAvatar />
-            <ExpandDownArrow />
-          </AvatarContainer>
-        </NavUser>
-      </NavInner>
-    </NavOuter>
-  );
-};
+          <NavDropDown label="How Civil works">
+            <NavLink href="https://civil.co/constitution/" target="_blank">
+              <NavLinkConstitutionText />
+            </NavLink>
+            <NavLink href="https://civil.co/about/" target="_blank">
+              <NavLinkAboutText />
+            </NavLink>
+            <NavLink href="https://civil.co/how-to-launch-newsroom/" target="_blank">
+              <NavLinkLaunchNewsroomText />
+            </NavLink>
+            <NavLink href="https://civil.co/white-paper/" target="_blank">
+              <NavLinkWhitePaperText />
+            </NavLink>
+          </NavDropDown>
+          <NavAccent>
+            <NavLink to="/dashboard">
+              <NavLinkDashboardText />
+            </NavLink>
+          </NavAccent>
+          <NavUser onClick={ev => this.toggle()}>
+            <CvlContainer>
+              <CvlToken />
+              <span>
+                <UserCvlBalance>{this.props.balance}</UserCvlBalance>
+                <UserCvlVotingBalance>{this.props.votingBalance}</UserCvlVotingBalance>
+              </span>
+            </CvlContainer>
+            <AvatarContainer>
+              <UserAvatar />
+              <Arrow isOpen={this.state.isOpen} />
+            </AvatarContainer>
+          </NavUser>
+        </NavInner>
+        {this.state.isOpen ? (
+          <NavDrawerComponent
+            balance={this.props.balance}
+            votingBalance={this.props.votingBalance}
+            userAccount={this.props.userAccount}
+            userChallengesVotedOnCount={this.props.userChallengesVotedOnCount}
+            userChallengesStartedCount={this.props.userChallengesStartedCount}
+            buyCvlUrl={this.props.buyCvlUrl}
+          />
+        ) : null}
+      </NavOuter>
+    );
+  }
+
+  private toggle(): void {
```

Comment:
If you change this method to use an arrow function, you won't need to bind the scope in the constructor and you can just pass the method as the `onClick` prop, instead of using a arrow method there.

Also, the arrow function seems to be the preferred best practice for React these days.

```
private toggle: void = () => {}
```